Mathlib/Data/Real/Basic.lean

```diff
@@ -607,3 +607,11 @@ def IsPowMul {R : Type*} [Pow R ℕ] (f : R → ℝ) :=
 def RingHom.IsBoundedWrt {α : Type*} [Ring α] {β : Type*} [Ring β] (nα : α → ℝ) (nβ : β → ℝ)
     (f : α →+* β) : Prop :=
   ∃ C : ℝ, 0 < C ∧ ∀ x : α, nβ (f x) ≤ C * nα x
+
+/-- If `x` is multiplicative with respect to `f`, then so is any `x^n`. -/
```

Comment:
It would be interesting to have this accept a `@[to_additive]` tag, since the following works:
```
-- theorem fae {R : Type*} [CommRing R] (f : R → ℝ) {x : R}
--     (hx : ∀ y : R, f (x + y) = f x + f y) : ∀ (n : ℕ) (y : R), f (n * x + y) = n * f x + f y := by
--   intro n
--   induction n with
--   | zero => intro y; rw [Nat.cast_zero, zero_mul, Nat.cast_zero, zero_mul, zero_add, zero_add]
--   | succ n hn => intro y; rw [Nat.cast_succ, Nat.cast_succ, 
--     add_mul, one_mul, add_mul, one_mul, add_assoc, hn (x + y), hx, ← add_assoc]
```
but the proof is slightly different and cannot be stupidly ported.

Reply:
Do you think this could be done in a different PR?

Reply:
Sure!